lightning-network-book.tex

```diff
@@ -111,12 +111,12 @@ \section{Introduction to Electronic Cash}
 The recipient has no chance of knowing that the sender actually deleted the coin.
 If the sender did not delete the coin the sender would be able to spend the coin again by copying the information to another person.
 This process is called double spending.
-Obviously double spending sabotages the property that cash should act as a store of value. As the supply would be infinite a single coin would always be worthless. 
+Obviously, double spending sabotages the property that cash should act as a store of value. When the supply is infinite, a single coin becomes worthless.
 
 We can conclude that electronic cash is only useful if we solve the problem of double spending electronic coins and create a limited supply of electronic coins.
 
 One obvious solution is a central authority or custodian who understands who owns which coins and who allows cash transactions only if the sender has enough cash.
-While this solution works and is already implemented by traditional banks it does not solve our desire that we carry around our electronic coins in our devices and - even worse - we still need to trust third-party service with all the risks that we have experienced with the collapse of Lehman Brothers.
+While this solution works and is already implemented by traditional banks, it does not solve our desire that we carry around our electronic coins in our devices and - even worse - we still need to trust third-party service with all the risks that we have experienced with the collapse of Lehman Brothers.
 
 Luckily in late 2008 a person or group under the pseudonym Satoshi Nakamoto published the Bitcoin paper.
```

Comment:
add link to white paper?

Reply:
will happen at some point in time but it is not clear that the paper is mentioned here for the first time